ui/DesignSystem/Component/TrackedRemoteListItem.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  import { Avatar, Flex } from "../../DesignSystem/Primitive";
+  import Badge from "./Badge.svelte";
+  import Tooltip from "./Tooltip.svelte";
+
+  import TrackToggle from "./TrackToggle.svelte";
+  import { BadgeType } from "../../src/badge.ts";
+  import { CSSPosition } from "../../src/style";
+  import { PeerType, removeRemote, Role } from "../../src/project";
+
+  export let peer = null;
+  export let projectId = null;
+  export let projectName = null;
+
+  const unfollowRemote = () => {
+    removeRemote(projectId, peer.peerId);
```

Comment:
Should be a dispatch this component should not do any data manipulation, ought to be responsibility of the modal.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1014/commits/1385636c46b3c1812fd6233db5b8731b57143038